src/gsi/chemmod.f90

```diff
@@ -45,7 +56,11 @@ module chemmod
   public :: code_pm10_ncbufr,code_pm10_anowbufr
 
   public :: l_aoderr_table
-  public :: laeroana_gocart
+
+  public :: laeroana_gocart,laeroana_fv3cmaq,laeroana_fv3smoke,crtm_aerosol_model,crtm_aerosolcoeff_format,crtm_aerosolcoeff_file, &
```

Comment:
The same suggestion:  more documentation would be very helpful for its' readability.  I  know the documentation to key options (and which is no so easy to figure out )  has not been stressed as much as I expect.  Even myself would, quite sometimes ,forgot to add the doc that should be.  So, except for a few parameter very often used in the GSI flow as I mentioned in another comment,  others would be left the developer to decide if he has the time to add more documentation.